assets/js/modules/analytics-4/datastore/audiences.test.js

```diff
@@ -304,6 +307,107 @@ describe( 'modules/analytics-4 audiences', () => {
 			} );
 		} );
 
+		describe( 'maybeSyncAvailableAudiences', () => {
+			it( 'should call syncAvailableAudiences if the availableAudiencesLastSyncedAt setting is undefined', async () => {
+				fetchMock.post( syncAvailableAudiencesEndpoint, {
+					body: availableAudiencesFixture,
+					status: 200,
+				} );
+
+				await registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.maybeSyncAvailableAudiences();
+
+				expect( fetchMock ).toHaveFetchedTimes( 1 );
+				expect( fetchMock ).toHaveFetched(
+					syncAvailableAudiencesEndpoint
+				);
+			} );
+
+			it( 'should not call syncAvailableAudiences if the availableAudiencesLastSyncedAt is within the last hour', async () => {
```

Comment:
Minor typo/grammatical error:

```suggestion
			it( 'should not call syncAvailableAudiences if the availableAudiencesLastSyncedAt setting is within the last hour', async () => {
```